src/app/wizard/step/provider-settings/provider/basic/openstack/template.html

```diff
@@ -49,11 +49,13 @@
     </mat-button-toggle>
   </mat-button-toggle-group>
 
-  <ng-container *ngIf="credentialsType === CredentialsType.Default">
+  <ng-container *ngIf="credentialsType === CredentialsType.Default"
+                [disabled]="isPresetSelected">
```

Comment:
The `[disabled]="isPresetSelected"` has no meaning here as this is a special angular element.

https://angular.io/api/core/ng-container

Reply:
Remanent from the dual support for credentials, will remove this.